### PR TITLE
feat: add howto change creditcard

### PR DESCRIPTION
**Pull request by andersc2022**
Guide how to change to new creditcard

### CODE REVIEW

**Comment by fghaas** [comment]
@andersc2022 Please don't "rebase" with a merge. I know GitHub does this by default in its web UI, but it's utterly silly and I haven't yet found a way to turn it off. Please fix your branch as follows:

```bash
git checkout main
git pull
git checkout Change-Credit-Card
git reset --hard 9a7a99c93c25d197df4605781f9c3b7290b424d3
git rebase main
git push --force-with-lease
```

File: docs/howto/account-billing/change-credit-card.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Changing to new credit card
+
+You can only change your credit card when you _pay a new invoice_ via [{{gui}}](https://{{gui_domain}})
+
+## Prerequisites
+
+You must be logged in at your [Cleura 
+Cloud](https://{{gui_domain}}) account to switch credit card.
```
Comment (fghaas) [inline docs/howto/account-billing/change-credit-card.md:8]: Say "switch credit cards" or "switch to a new credit card", but not "switch credit card".

File: docs/howto/account-billing/change-credit-card.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## Change credit card
+
+Navigate to the Dashboard list in the left-hand menu and click on _Invoices_.
+
+![White calculator and the word invoice on orange background.](assets/dashboardinvoice.png)
+
+Now you will see your invoice specifications. Click on the _circle with three dots_ beside the invoice you wanna pay.
```
Comment (fghaas) [inline docs/howto/account-billing/change-credit-card.md:16]: Use "want to" or "would like to", not "wanna".

File: docs/howto/account-billing/change-credit-card.md (hunk 1 above)
Comment (fghaas) [inline docs/howto/account-billing/change-credit-card.md:8]: Say "switch credit cards" or "switch to a new credit card", but not "switch credit card".

File: docs/howto/account-billing/change-credit-card.md (hunk 2 above)
Comment (fghaas) [inline docs/howto/account-billing/change-credit-card.md:16]: Use "want to" or "would like to", not "wanna".

**Review by fghaas — COMMENTED** [review]
This looks mostly good now. I have two small suggestions.

Also, please don't use blue bars to hide confidential information. We have been using blurring/pixelation in other screenshots. @colder-is-better, at your convenience can you please give Anders some hints on what options you use in the GIMP here, just so we're all on the same page?

**Comment by colder-is-better** [comment]
I have to admit, I like those blue bars on @andersc2022's screenshots :) Personally, though, I prefer hiding sensitive info on screenshots by pixelating rectangular areas. Also, I don't use GIMP. For light editing screenshots, I use SnagIt. It's not free, but I've gotten used to it, and by now, it just Works For Me (TM).

**Review by fghaas — APPROVED** [review]
Took the liberty to rebase this on current `main`, and made a couple of tiny changes (whitespace fixes, punctuation). Will proceed to merge now. Thanks @andersc2022!

**Pull request merged**